time_tracker_api/time_entries/time_entries_namespace.py

```diff
@@ -35,7 +35,7 @@
         description='The id of the selected activity',
         example=faker.uuid4(),
     ),
-    'description': fields.String(
```

Comment:
If you change this description better change it in the other namespaces.

Reply:
makes sense, I will